### PR TITLE
corrected  timeout and changed underneath test

### PR DESCRIPTION
**Pull request by miyadav**
@huali9 once your change on our test repo is done and merged , will test this. 
cc @sunzhaohua2

### CODE REVIEW

**Comment by miyadav** [comment]
/pj-rehearse pull-ci-openshift-cluster-control-plane-machine-set-operator-release-4.18-regression-clusterinfra-aws-ipi-cpms-allcases

**Comment by miyadav** [comment]
/pj-rehearse pull-ci-openshift-cluster-control-plane-machine-set-operator-release-4.17-regression-clusterinfra-aws-ipi-cpms-allcases

**Comment by miyadav** [comment]
/pj-rehearse pull-ci-openshift-cluster-control-plane-machine-set-operator-release-4.18-regression-clusterinfra-aws-ipi-cpms-allcases

**Comment by miyadav** [comment]
/pj-rehearse pull-ci-openshift-cluster-control-plane-machine-set-operator-release-4.17-regression-clusterinfra-aws-ipi-cpms-allcases

**Comment by miyadav** [comment]
@JoelSpeed , [test](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/55907/rehearse-55907-pull-ci-openshift-cluster-control-plane-machine-set-operator-release-4.18-regression-clusterinfra-aws-ipi-cpms-allcases/1828260476560084992) looks good all operators are happy , seems like statuses of 4.17 and 4.18 are interchanged as 4.17 is showing as passed and 4.18 as failed..

**Comment by JoelSpeed** [comment]
Hmm, it seems the 4.18 hit a timeout and that's why it failed, even though the tests pass, teardown perhaps took too long?

And I'm not sure why the 4.17 didn't fail, it should have done really, which is odd, lets try those both again

/pj-rehearse pull-ci-openshift-cluster-control-plane-machine-set-operator-release-4.17-regression-clusterinfra-aws-ipi-cpms-allcases

/pj-rehearse pull-ci-openshift-cluster-control-plane-machine-set-operator-release-4.18-regression-clusterinfra-aws-ipi-cpms-allcases

**Comment by JoelSpeed** [comment]
/pj-rehearse pull-ci-openshift-cluster-control-plane-machine-set-operator-release-4.17-regression-clusterinfra-aws-ipi-cpms-allcases

/pj-rehearse pull-ci-openshift-cluster-control-plane-machine-set-operator-release-4.18-regression-clusterinfra-aws-ipi-cpms-allcases

**Comment by huali9** [comment]
For this run https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/55907/rehearse-55907-pull-ci-openshift-cluster-control-plane-machine-set-operator-release-4.18-regression-clusterinfra-aws-ipi-cpms-allcases/1828260476560084992, all CPMS cases passed, but it still report error, seems the timeout should be increased, because it shows `Test started yesterday at 10:38 AM failed after 7h43m24s` in the run, but the timeout is `timeout: 7h30m0s`, seems not enough.

For this run https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/55907/rehearse-55907-pull-ci-openshift-cluster-control-plane-machine-set-operator-release-4.17-regression-clusterinfra-aws-ipi-cpms-allcases/1828260519136464896, although it report passed, but one case (53323) failed. `Aug 27 07:10:48.504 W ns/openshift-machine-api machine/ci-op-jxr98gkm-45d3f-9hsrg-master-trh6z-0 ci-op-jxr98gkm-45d3f-9hsrg-master-trh6z-0: reconciler failed to Create machine: failed to launch instance: error launching instance: The placement group 'pgpartition3' is unknown.` It still complains the placement group is unknown although it run in us-east-2 region, so I think it doesn't use our test account for the run. I think we should figure out what account it's using and create the placement group in advance, otherwise it won't run.

**Comment by miyadav** [comment]
@huali9 , 4.18 and 4.17 both must be using same account ,timeout we can increase and see.

**Comment by huali9** [comment]
> @huali9 , 4.18 and 4.17 both must be using same account ,timeout we can increase and see.

Right, but the region is random, and for the 4.18 run https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/55907/rehearse-55907-pull-ci-openshift-cluster-control-plane-machine-set-operator-release-4.18-regression-clusterinfra-aws-ipi-cpms-allcases/1828260476560084992, all CPMS cases passed, because it run in us-west-2 region, so it didn't hit the issue.

**Comment by miyadav** [comment]
/pj-rehearse pull-ci-openshift-cluster-control-plane-machine-set-operator-release-4.17-regression-clusterinfra-aws-ipi-cpms-allcases

/pj-rehearse pull-ci-openshift-cluster-control-plane-machine-set-operator-release-4.18-regression-clusterinfra-aws-ipi-cpms-allcases

**Comment by miyadav** [comment]
> > @huali9 , 4.18 and 4.17 both must be using same account ,timeout we can increase and see.
> 
> Right, but the region is random, and for the 4.18 run https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/55907/rehearse-55907-pull-ci-openshift-cluster-control-plane-machine-set-operator-release-4.18-regression-clusterinfra-aws-ipi-cpms-allcases/1828260476560084992, all CPMS cases passed, because it run in us-west-2 region, so it didn't hit the issue.

acked

**Comment by miyadav** [comment]
/pj-rehearse pull-ci-openshift-cluster-control-plane-machine-set-operator-release-4.18-regression-clusterinfra-aws-ipi-cpms-allcases

**Comment by miyadav** [comment]
/pj-rehearse pull-ci-openshift-cluster-control-plane-machine-set-operator-release-4.18-regression-clusterinfra-aws-ipi-cpms-allcases

**Comment by miyadav** [comment]
/pj-rehearse pull-ci-openshift-cluster-control-plane-machine-set-operator-release-4.17-regression-clusterinfra-aws-ipi-cpms-allcases

**Comment by miyadav** [comment]
@JoelSpeed @elmiko We can merge this now.

File: ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-main.yaml [hunk 1]
```diff
@@ -280,7 +280,7 @@ tests:
     env:
       TEST_FILTERS: NonHyperShiftHOST&;cpms;CPMS
       TEST_SCENARIOS: Cluster_Infrastructure
-      TEST_TIMEOUT: "1200"
+      TEST_TIMEOUT: "90"
```
Comment (JoelSpeed) [inline ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-main.yaml:283]: What is the unit here, we are going from 1200 to 90 which seems like a significant reduction?

Comment (miyadav) [reply ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-main.yaml:283]: Yeah , Joel , I updated title , it was misleading , initially while reviewing we thought it is `seconds` and is less , hence causing failures , realized it wasn't that only , but `PlacementGroups` tests added which were causing it and the time is in `mins` which was wrongly used as `1200` instead should be `90`
So only change in this repo was time, the tests was changed on another repo fixing the failures here.

**Pull request merged**